ui/pages/routes/routes.component.js

```diff
@@ -307,7 +305,7 @@ export default class Routes extends Component {
 
     const isInitializing = Boolean(
       matchPath(location.pathname, {
-        path: process.env.ONBOARDING_V2 ? ONBOARDING_ROUTE : INITIALIZE_ROUTE,
```

Comment:
I might be missing something... can we also remove the `INITIALIZE_ROUTE`?

Reply:
I'll do this in a separate PR, where I remove all the old components and routes.